style.css

```diff
@@ -0,0 +1,24 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;700&display=swap");
```

Comment:
razmisli treba li nam import ovdje https://www.thoughtco.com/difference-between-important-and-link-3466404